starter-code/scripts/articleView.js

```diff
@@ -75,35 +75,50 @@ articleView.setTeasers = function() {
 
 articleView.initNewArticlePage = function() {
   // TODO: Ensure the main .tab-content area is revealed. We might add more tabs later or otherwise edit the tab navigation.
-
-
+    $('.tab-content').show();
   // TODO: The new articles we create will be copy/pasted into our source data file.
   // Set up this "export" functionality. We can hide it for now, and show it once we have data to export.
-
+  $('#article-export').hide();
   $('#article-json').on('focus', function(){
     this.select();
   });
 
-  // TODO: Add an event handler to update the preview and the export field if any inputs change.
 
+  // TODO: Add an event handler to update the preview and the export field if any inputs change.
+    $('#newArticle').on('change', function() {
+      articleView.create();
+      //update .articles
+    })
 };
 
 articleView.create = function() {
   // TODO: Set up a var to hold the new article we are creating.
   // Clear out the #articles element, so we can put in the updated preview
 
+    var newRawData = {};
+      newRawData.title = $('#title').val();
+      newRawData.author = $('#author').val();
+      newRawData.authorUrl = $('#authorUrl').val();
```

Comment:
`$('#authorUrl').val()` will return undefined because the id in new.html is `AuthorUrl`
don't forget JS is case sensitive